react/src/components/atoms/buttons/ButtonWithIcon/style.scss

```diff
@@ -1,10 +1,54 @@
 @import 'global';
+@import '00-base/mixins';
 @import "01-atoms/button-search";
 
+$colors: (
+  "c-highlight": $c-highlight,
+  "c-primary": $c-primary,
+  "c-primary-alt": $c-primary-alt,
+  "c-black": $c-black,
+  "c-gray-dark": $c-gray-dark,
+  "c-white": $c-white
+);
+@mixin ma-button-icon-secondary ($color: $c-primary-alt) {
```

Comment:
can you move these @mixins into mixins folder? just so it stays consistent with _buttons.scss

maybe see if these make sense to be somehow combined in the _ma-button mixin (just a thought)

Reply:
removed mixins here.